src/app/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { User } from '../interfaces';
+import { Router } from '@angular/router';
+import { TableService } from '../table-service';
+
+@Component({
+  selector: 'app-sign-up',
+  templateUrl: './sign-up.component.html',
+  styleUrls: ['./sign-up.component.scss'],
+})
+export class SignUpComponent implements OnInit {
+  user: User | any = {};
```

Comment:
why this needed?

Reply:
I give it to function "register" as a parameter

Reply:
ok good